utah-design-system-website/src/react/components/websiteContent/library/components/widgetsIndicators/badges/BadgesDocumentation.jsx

```diff
@@ -145,18 +188,33 @@ export function BadgesDocumentation() {
                     </TableHead>
                     <TableBody>
                       <TableRow>
-                        <TableCell><code className="primary-color">appearance</code></TableCell>
+                        <TableCell><code className="primary-color">children</code></TableCell>
+                        <TableCell><code>react node</code></TableCell>
```

Comment:
We may want to standardize on what type `children` shows in documentation props. Some say `node`, some say `react node`, some say `React.ReactNode`, or `React Node`.

Reply:
`React.ReactNode`